packages/alfa-aria/test/name.spec.tsx

```diff
@@ -1167,7 +1182,7 @@ test(`.from() determines the name of an <input type="image"> element`, (t) => {
   t.deepEqual(Name.from(input, device).toJSON(), {
     type: "some",
     value: {
-      value: "Submit",
+      value: "Submit Query",
```

Comment:
How come this didn't fail before? 😮

Reply:
I don't know and at this point I'm too afraid to dig 🙈